<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(56, 57, 52);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1261</width>
     <height>751</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>font/logo.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="exit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:  transparent;
border:none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="application-exit">
          <normaloff>font/退出.png</normaloff>font/退出.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>1020</width>
          <height>0</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page_5">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>-1</y>
            <width>1001</width>
            <height>641</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QGroupBox
{
	
	background-color: transparent;
	font: 12pt &quot;Microsoft YaHei UI&quot;;
	color: rgb(131, 131, 131);
	border-radius:10px;
	border:2px solid gray;
}</string>
          </property>
          <property name="title">
           <string>视频流导入</string>
          </property>
          <widget class="QComboBox" name="file_list">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>270</y>
             <width>651</width>
             <height>40</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="mouseTracking">
            <bool>false</bool>
           </property>
           <property name="tabletTracking">
            <bool>false</bool>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox{
	color: rgb(255, 255, 255);
	font: 12pt &quot;Microsoft YaHei UI&quot;;
	background-color:transparent;
	border:2px solid gray;
	border-radius:15px;
}
QComboBox::drop-down 
{
	subcontrol-origin: padding;
	subcontrol-position: top right;
	width: 50px;
}
QComboBox::down-arrow
{
	/*image:url(:/font/down-arrow.png);*/
}
  QComboBox QAbstractItemView{
  border:2px solid gray;
  border-radius:3px;
	background-color:rgb(56, 57, 52);
    outline: 0px;  
}

QComboBox QAbstractItemView::item{
	height:36px;
	color:white;
	padding-left:0px;
}

QComboBox QAbstractItemView::item:selected{
  background-color:#97BE15;
  color:#ffffff;
  border:2px solid gray;
  border-radius:15px;
}</string>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="duplicatesEnabled">
            <bool>false</bool>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>111111111111111111111111111111</string>
            </property>
           </item>
          </widget>
          <widget class="QPushButton" name="open_file">
           <property name="geometry">
            <rect>
             <x>840</x>
             <y>260</y>
             <width>60</width>
             <height>60</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:  transparent;
border:none;</string>
           </property>
           <property name="icon">
            <iconset theme="face-laugh">
             <normaloff>font/搜索.png</normaloff>font/搜索.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
          </widget>
          <widget class="QPushButton" name="stream_import">
           <property name="geometry">
            <rect>
             <x>890</x>
             <y>540</y>
             <width>60</width>
             <height>60</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:  transparent;
border:none;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="accessories-calculator">
             <normaloff>font/进入.png</normaloff>font/进入.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="page_6">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>-1</y>
            <width>1001</width>
            <height>641</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QGroupBox
{
	
	font: 12pt &quot;Microsoft YaHei UI&quot;;
	color: rgb(131, 131, 131);
	border-radius:10px;
	border:2px solid gray;
}</string>
          </property>
          <property name="title">
           <string>结果预览</string>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget_3">
           <property name="geometry">
            <rect>
             <x>9</x>
             <y>19</y>
             <width>981</width>
             <height>611</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="1" column="0">
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="annotated_image">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>960</width>
                  <height>540</height>
                 </size>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="1">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="2" column="1">
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="stream_reload">
                <property name="styleSheet">
                 <string notr="true">background-color:  transparent;
border:none;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>font/左箭头.png</normaloff>font/左箭头.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>40</width>
                  <height>40</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="styleSheet">
           <string notr="true">QGroupBox
{
	
	font: 12pt &quot;Microsoft YaHei UI&quot;;
	color: rgb(131, 131, 131);
	border-radius:10px;
	border:2px solid gray;
}</string>
          </property>
          <property name="title">
           <string>结果显示</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
